src/support/pagelocker.h

```diff
@@ -145,9 +145,9 @@ class LockedPageManager : public LockedPageManagerBase<MemoryPageLocker>
     static void CreateInstance()
     {
         // Using a local static instance guarantees that the object is initialized
-        // when it's first needed and also deinitialized after all objects that use
+        // when it's first needed and also uninitialized after all objects that use
```

Comment:
"destructed" (this is C++ jargon; I know the more common usage would be "destroyed", but "destructed" here makes it clearer what is meant).